isis/tests/FunctionalTestsExplode.cpp

```diff
@@ -0,0 +1,97 @@
+#include <QFileInfo> 
+#include <QString>
+#include <QTemporaryDir>
+#include <QTextStream>
+
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "explode.h"
+
+#include "gtest/gtest.h"
+
+using namespace std;
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/explode.xml").expanded();
+
+/**
+   * Explode Case01 Test
+   * 
+   * Export (i.e. Explode) each band of an input
+   * cube with two bands into individual cubes.
+   * 
+   * INPUT: isisTruth.cub
+   * 
+   * OUTPUT: 1) explTruth1.band0001.cub
+   *         2) explTruth1.band0002.cub
+   */
+TEST(Explode, ExplodeCase01) {
```

Comment:
Is it possible to use the DefaultCube test fixture instead of including the isisTruth.cub?